### DISCUSSION TITLE
Validation W3C

### DISCUSSION THREAD

**Discussion by AllanNobi** [AmauriC/tarteaucitron.js#1082] [Q&A]
Bonjour,

Pouvez-vous modifié les attributs présents sur les DIV (avant chargement des iframes) pour les remplacer par des data-<nomattribut> (videoID, showinfo, autoplay,embedtype, width, height...) ?
Exemple pour Calaméo : <div class="calameo-canvas" data-id="bkcode" width="width" height="height"></div>
-> <div class="calameo-canvas" data-id="bkcode" data-width="width" data-height="height"></div>

Exemple pour Dailymotion : <div class="dailymotion_player" videoID="video_id" width="width" height="height" showinfo="1" autoplay="0" embedType="video"></div>
-> <div class="dailymotion_player" data-videoid="video_id" data-width="width" data-height="height" data-showinfo="1" data-autoplay="0" data-embedType="video"></div>

Et lors du chargement des iframes, ne pas charger les attributs obsolètes.
Exemple : frameborder="0" et scrolling="no"

**Comment by AmauriC** [comment]
Bonjour,

Pour Dailymotion, Youtube, Vimeo, Maps, .. vous pouvez déjà ajouter le `data-` devant le nom de l'attribut, tarteaucitron comprend les 2 types.
